network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_table_block.html

```diff
@@ -7,7 +7,7 @@
     {% comment %}
       Modified table template comes from "table_block/blocks/table.html"
     {% endcomment %}
-    <table class="table">
+    <table class="table">{% if column_header %}gary gary doug{% endif %}
```

Comment:
remove this? 😃 

```suggestion
    <table class="table">
```

Reply:
Haha was testing a thing. Yesss thank you! That spilled passed my `git diff` woops 😬